.github/workflows/firmwares.yml

```diff
@@ -56,6 +56,7 @@ jobs:
         run: | 
           echo "[patch.crates-io]" >> solo2/runners/lpc55/Cargo.toml
           echo "trussed = { path = '../../../trussed' }" >> solo2/runners/lpc55/Cargo.toml 
+          echo "oath-authenticator = { git = 'https://github.com/trussed-dev/oath-authenticator', rev = '20d9018a567e436f9037d38985f6c0f96d9921fe' }" >> solo2/runners/lpc55/Cargo.toml
```

Comment:
Fix before merge?

Reply:
It’s already upstreamed.  Should we just remove the patch here and allow failures on the firmwares build?